--- origsrc/gtkaml-0.2.9.1/configure.ac	2009-11-13 16:13:38.000000000 -0600
+++ src/gtkaml-0.2.9.1/configure.ac	2009-11-14 21:25:26.915561100 -0600
@@ -13,9 +13,6 @@ case $host in
 *-*-mingw32* | *-*-windows)
 		EXTRA_CFLAGS="-mno-cygwin -mwindows -mms-bitfields -fno-strict-aliasing "	
 		EXTRA_LDFLAGS="-export-all-symbols -mwindows -no-undefined -mms-bitfields" ;;
-*-*-cygwin*)
-		EXTRA_CFLAGS="-fno-strict-aliasing"
-		EXTRA_LDFLAGS="-export-all-symbols -mms-bitfields";;
 esac
 AC_SUBST(EXTRA_CFLAGS)
 AC_SUBST(EXTRA_LDFLAGS)
