From r824

--- origsrc/gtkaml-0.4.3/configure.ac	2011-11-19 22:08:52.463612100 -0600
+++ src/gtkaml-0.4.3/configure.ac	2011-11-19 22:10:38.242662300 -0600
@@ -39,8 +39,12 @@ PKG_CHECK_MODULES([LIBXML2], libxml-2.0)
 AC_SUBST(LIBXML2_CFLAGS)
 AC_SUBST(LIBXML2_LIBS)
 
-PKG_CHECK_MODULES(VALA, libvala-0.12,[VALA_PKG="libvala-0.12"],[PKG_CHECK_MODULES(VALA, vala-0.10, [VALA_PKG="vala-0.10"])])
-AM_CONDITIONAL([VALA_LATEST], [test $VALA_PKG = libvala-0.12])
+PKG_CHECK_MODULES(VALA, libvala-0.14,[VALA_PKG="libvala-0.14"],
+	[PKG_CHECK_MODULES(VALA, vala-0.12, [VALA_PKG="vala-0.12"],
+		[PKG_CHECK_MODULES(VALA, vala-0.10, [VALA_PKG="vala-0.10"])
+	])
+])
+AM_CONDITIONAL([VALA_LATEST], [test $VALA_PKG = libvala-0.14])
 
 AC_SUBST(VALA_PKG)
 AC_SUBST(VALA_CFLAGS)
--- origsrc/gtkaml-0.4.3/src/GtkamlCompiler.vala	2011-03-13 16:31:11.000000000 -0500
+++ src/gtkaml-0.4.3/src/GtkamlCompiler.vala	2011-11-19 22:13:00.290787000 -0600
@@ -67,7 +67,9 @@ class Gtkaml.Compiler {
 	static bool deprecated;
 	static bool experimental;
 	static bool experimental_non_null;
+#if !VALA_0_14
 	static bool disable_dbus_transformation;
+#endif
 	static bool disable_warnings;
 	static string cc_command;
 	[CCode (array_length = false, array_null_terminated = true)]
@@ -126,7 +128,9 @@ class Gtkaml.Compiler {
 		{ "disable-warnings", 0, 0, OptionArg.NONE, ref disable_warnings, "Disable warnings", null },
 		{ "fatal-warnings", 0, 0, OptionArg.NONE, ref fatal_warnings, "Treat warnings as fatal", null },
 		{ "enable-experimental-non-null", 0, 0, OptionArg.NONE, ref experimental_non_null, "Enable experimental enhancements for non-null types", null },
+#if !VALA_0_14
 		{ "disable-dbus-transformation", 0, 0, OptionArg.NONE, ref disable_dbus_transformation, "Disable transformation of D-Bus member names", null },
+#endif
 		{ "cc", 0, 0, OptionArg.STRING, ref cc_command, "Use COMMAND as C compiler command", "COMMAND" },
 		{ "Xcc", 'X', 0, OptionArg.STRING_ARRAY, ref cc_options, "Pass OPTION to the C compiler", "OPTION..." },
 		{ "dump-tree", 0, 0, OptionArg.FILENAME, ref dump_tree, "Write code tree to FILE", "FILE" },
@@ -181,7 +185,9 @@ class Gtkaml.Compiler {
 		context.deprecated = deprecated;
 		context.experimental = experimental;
 		context.experimental_non_null = experimental_non_null;
+#if !VALA_0_14
 		context.dbus_transformation = !disable_dbus_transformation;
+#endif
 		context.report.enable_warnings = !disable_warnings;
 		context.report.set_verbose_errors (!quiet_mode);
 		context.verbose_mode = verbose_mode;
@@ -304,6 +310,7 @@ class Gtkaml.Compiler {
 		}
 
 		if (context.profile == Profile.GOBJECT) {
+#if !VALA_0_14
 			if (context.has_package ("dbus-glib-1")) {
 				if (!context.deprecated) {
 					Report.warning (null, "D-Bus GLib is deprecated, use GDBus");
@@ -312,6 +319,9 @@ class Gtkaml.Compiler {
 			} else {
 				context.codegen = new GDBusServerModule ();
 			}
+#else
+			context.codegen = new GDBusServerModule ();
+#endif
 		} else if (context.profile == Profile.DOVA) {
 			context.codegen = new DovaErrorModule ();
 		} else {
